src/SelectMulti.vue

```diff
@@ -165,7 +165,7 @@
 					 * emits the current user-provided search string,
 					 * primarily useful for making autocomplete calls
 					 */
-					this.$emit('searchChange', this.inputValue)
+					this.$emit('searchChange', this.inputValue, this.filteredOptions)
```

Comment:
Yup, sounds good! It'll get the most recent `this.filteredOptions` because `this.inputValue` has already been updated, correct?

I definitely prefer this approach of adding a second argument rather than emitting an object, because:
1. It's non-breaking
2. We don't have to worry about this second argument in most of our other cases